packages/pointer_interceptor/pointer_interceptor_web/lib/pointer_interceptor_web.dart

```diff
@@ -18,6 +18,8 @@ class PointerInterceptorWeb extends PointerInterceptorPlatform {
 
   // Slightly modify the created `element` (for `debug` mode).
   void _onElementCreated(Object element) {
+    (element as web.HTMLElement).style.width = '100%';
```

Comment:
Can't you use a cascade to avoid 3x a cast?

```suggestion
    (element as web.HTMLElement)
      ..style.width = '100%'
      ..style.height = '100%'
      ..style.backgroundColor = 'rgba(255, 0, 0, .5)';
```